aw/ViewControllers/Reaches/Filters/FilterDistanceViewController.swift

```diff
@@ -56,9 +58,21 @@ class FilterDistanceViewController: UIViewController {
     func showDistance() {
         slider.value = distance
 
+        if distance == 0 {
+            toggleViews(false)
+        }
+
         displayLocation()
     }
 
+    func toggleViews(_ show: Bool) {
```

Comment:
Minor: this function name is a bit ambiguous, since there's a toggle and you're also toggling views on and off.